src/components/testimonial/Testimonial.css

```diff
@@ -0,0 +1,121 @@
+.testimonial-card {
+  display: flex;
+  flex-direction: column;
+  border-radius: .3rem;
+  flex-basis: 20rem;
+  padding: 1rem;
+  max-width: 20rem;
+  flex-grow: 1;
+  row-gap: .7rem;

```

Comment:
Ik denk dat deze ook groter mag! En het is een goed idee om ook column-gap erbij te zetten, gezien de video's over twee rijen verspreid zullen staan!